server/middleware/searchHandler.js

```diff
@@ -43,12 +43,14 @@ async function handleSearch(req, res, next) {
             .map(bindDisplayElements(fromStaticList)));
         const { workstackTypeColumns } = await req.listService.fetch('S_SYSTEM_CONFIGURATION');
         // using DEFAULT columns for search
-        const workstackColumnsForSearch = workstackTypeColumns[0].workstackColumns;
+        const workstackTypeForSearch = workstackTypeColumns.find(
+            item => item.workstackType === 'DEFAULT'
```

Comment:
✅

Reply:
Is there a situation in which we will not find a `DEFAULT` workstack? Previously we'd just pick the top workstack, and so this would only break if there were no workstacks at all. Do we need a fallback in case `DEFAULT` isn't found? Or is `{columns: null}` handled elsewhere?

Reply:
We have two systems, one of them (WCS) didn't have the 'DEFAULT' workstack defined which I have fixed. So the simple answer is no, all systems have and should have 'DEFAULT' workstack defined otherwise this bug will come back :)

Reply:
I wonder if it's worth throwing an error or asserting that assumption some way, so that if we add a third system and don't add a `DEFAULT` it's very obvious. Otherwise, :shipit: